apps/src/code-studio/components/ImagePicker.jsx

```diff
@@ -26,40 +27,48 @@ export default class ImagePicker extends React.Component {
     useFilesApi: PropTypes.bool,
     soundPlayer: PropTypes.object,
     disableAudioRecording: PropTypes.bool,
+    currentValue: PropTypes.string,
+    currentImageType: PropTypes.string,
     //For logging purposes
     projectId: PropTypes.string,
     elementId: PropTypes.string
   };
 
-  state = {mode: 'files'};
+  state = {mode: 'file'};
 
   getAssetNameWithPrefix = icon => {
     this.props.assetChosen(ICON_PREFIX + icon);
   };
 
-  setIconMode = () => this.setState({mode: 'icons'});
-
-  setFileMode = () => this.setState({mode: 'files'});
+  setMode = mode => {
+    this.setState({mode: mode});
+  };
 
   render() {
-    const isFileMode = this.state.mode === 'files';
     const styles = {
       root: {
         margin: '0 0 0 5px'
       },
       fileModeToggle: {
         float: 'left',
         margin: '0 20px 0 0',
-        fontFamily: isFileMode ? '"Gotham 5r"' : null,
-        color: isFileMode ? null : '#999',
+        fontFamily: this.state.mode === 'file' ? '"Gotham 5r"' : null,
```

Comment:
we reference the mode string in multiple places, so we should move these strings into a "type" like this:

https://github.com/code-dot-org/code-dot-org/blob/1102ea9d2fd09dcc0a9fcb42abf9ad40c26ef7b3/apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx#L12-L17

then you can reference it as `MODE.FILE`, and its clear what all of the possible modes are